x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -708,6 +708,9 @@ export class TaskRunner<
 
```

Comment:
As discussed yesterday, we should also include errorSource on the log happening at line 755 for consistency.

Reply:
Removed the additional log and just added the source type to the tags